qutebrowser/commands/runners.py

```diff
@@ -33,6 +34,12 @@
 from qutebrowser.utils import message, objreg, qtutils, usertypes, utils
 from qutebrowser.misc import split, objects
 
+MYPY = False
+if MYPY:
+    # pylint: disable=unused-import
+    from qutebrowser.mainwindow import tabbedbrowser
+    # pylint: enable=unused-import
+ReplacementFunction = typing.Callable[['tabbedbrowser.TabbedBrowser'], str]
```

Comment:
```suggestion
_ReplacementFunction = typing.Callable[['tabbedbrowser.TabbedBrowser'], str]
```

as it shouldn't be needed from outside of this module